sdk/servicebus/azure-servicebus/tests/async_tests/mgmt_tests/test_mgmt_queues_async.py

```diff
@@ -294,6 +294,8 @@ async def test_async_mgmt_queue_update_success(self, servicebus_namespace_connec
             queue_description.lock_duration = datetime.timedelta(seconds=13)
             queue_description.max_delivery_count = 14
             queue_description.max_size_in_megabytes = 3072
+            queue_description.forward_to = "sb://{}.servicebus.windows.net/{}".format(servicebus_namespace.name, queue_name)
+            queue_description.forward_dead_lettered_messages_to = "sb://{}.servicebus.windows.net/{}".format(servicebus_namespace.name, queue_name)
```

Comment:
emmm, would that work if we set `forward_to` and `forward_dead_lettered_messages_to` to self.
or do we need to create a new queue to test these two properties

Reply:
I had the same question, so I tested it in azure.  It's valid, although kinda nonsensical.  You can even create it in the portal, so it's good to go for the test and way less overhead than creating additional resources.